src/publish/fail.js

```diff
@@ -96,4 +96,7 @@ exports.default = async function fail({ context, github, inputs, Sentry }) {
     release: `${inputs.repo}@${inputs.version}`,
   });
   client.captureMessage(`Release failed: ${inputs.repo}`, "error", null, scope);
+
+  const session = Sentry.getCurrentHub().startSession({ status: "crashed" });
+  Sentry.getCurrentHub().endSession(session);
```

Comment:
```suggestion
  const session = Sentry.getCurrentHub().startSession({
    release,
    status: "crashed"
  });
  client.captureSession(session);
```

And move release from line 96 to `release` const so we can reuse it in both places. The same change applies to `success.js`.

We could use raw `new Session` here, but its not exported from `@sentry/node`.